front/src/app/api/tags/route.ts

```diff
@@ -0,0 +1,10 @@
+import { TagsData } from "@/services/prisma/TagsData";
+
+export const GET = async () => {
+    try {
+        const tags = await TagsData.getTags();
+        return Response.json({ tags });
```

Comment:
For all responses, instead return a NextResponse, so that you can include a status code